docs/developer-docs/daos/nns/proposal-lifecycle.mdx

```diff
@@ -0,0 +1,89 @@
+Moreover, the governance canister computes and stores additional information with each proposal.
+First, the governance canister looks at all neurons and creates an (empty) ballot for each eligible neuron that has a
+dissolve delay of at least 6 months. Thereby, each neuron's ballot stores the neuron's voting power in this moment.
+This means that at the time of the proposal's submission a neuron snapshot is taken and all neurons can vote on the proposal
+with the voting power that they have in this moment.
+
+:::info
+This results in the following, potentially unintuitive cases:
+* If a neuron is created after a proposal was submitted, it is not eligible to vote on this proposal.
+* If a neuron had a dissolve delay of less than 6 months when a proposal was submitted, it cannot vote on the proposal, even if the neuron's dissolve delay is increased again.
+* If a neuron had a dissolve delay of 6 months when a proposal was submitted, it can vote on the proposal during the full voting period, even if by the time of voting the neuron's dissolve delay decreased slightly below 6 months.
+
+:::
+
+The sum of all of the neurons' voting powers determines the total voting power associated with a given proposal.
+In addition to creating all ballots, when a new proposal is created, the proposer's ballot is set to a “yes” vote.
+This reflects that the proposal already has the support of the user submitting it.
+
+Moreover, each proposal has an associated _voting period_, which determines the period of time over which votes for
+ this proposal are accepted.
+
+### How to submit a proposal
+To submit a proposal, it is easiest to have a neuron that can be controlled on a command line tool.
+You can find concrete instructions how to submit a proposal of type `Motion` [here](https://wiki.internetcomputer.org/wiki/How-To:_Create_an_NNS_motion_proposal).
+
+## Proposal decision and wait-for-quiet
+After a proposal is submitted and added to the governance canister, other users who control neurons can [vote on the proposal](./proposal-voting.mdx).
+The governance canister stores a neuron's choice in the still empty ballot where the neuron's voting power is already recorded.
+In this way, the proposal always contains the information of the total voting power of "Yes" and "No" votes.
+
+A proposal can be _decided_ in two ways:
+
+1. **Absolute Majority before the voting period ends**: At any point, even before the voting period ends, if an absolute
+majority (more than half of the total voting power stored in the proposal) has voted Yes, then the proposal is adopted and if
+ an absolute majority has voted No, then the proposal is rejected.
+2. **Simple Majority at the voting period’s end**: When the voting period ends, if a simple majority (more than half of the
+cast votes) has voted Yes and the number of these Yes-votes constitute at least 3% of the total voting power, then the
+proposal is adopted. Otherwise, the proposal is rejected.
```

Comment:
```suggestion
1. **Absolute majority before the voting period ends**: At any point, even before the voting period ends, if an absolute majority (more than half of the total voting power stored in the proposal) has voted "Yes", then the proposal is adopted, and if an absolute majority has voted "No", then the proposal is rejected.
2. **Simple majority at the voting period’s end**: When the voting period ends, if a simple majority (more than half of the cast votes) has voted "Yes" and the number of these "Yes"-votes constitute at least 3% of the total voting power, then the proposal is adopted. Otherwise, the proposal is rejected.
```